### PR TITLE
Flesh out WEBIRC docs a bit

### PR DESCRIPTION
**Pull request by lol768**
Includes a bit more background information and some useful links.

### CODE REVIEW

File: docs/advanced/webirc.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
-KICL supports the WEBIRC command.
+*Please note that some server software refers to the WebIRC protocol as cgiirc in configuration files.*
+
+KICL supports the WEBIRC command which is used for indirect connections such that the IRC server is made aware of the user's own IP address. For example, consider a webchat client. Without WebIRC support, all connections made through the web client would appear to originate fromt the webserver's IP address. With WebIRC support, the (trusted) webchat client sends the following information on connect:
+
```
Comment (mbax) [inline docs/advanced/webirc.md]: sends the following info, and then instead states version support

File: docs/advanced/webirc.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+KICL supports version 1 of the specification, documented [here](https://kiwiirc.com/docs/webirc).
+
+* A password that is kept secret between the IRC server and the client requesting the spoofing.
+* The name of the client doing the proxying and requesting the spoof (e.g. this is 'qwebirc' for Iris and 'KiwiIRC' for Kiwi IRC).
```
Comment (mbax) [inline docs/advanced/webirc.md]: KICL doesn't send the client name?

File: docs/advanced/webirc.md [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+The code sample below shows how to configure KICL to send a WEBIRC command on connect:
 
 ```
 InetAddress ip = ...; // User's IP address
 Client client = Client.builder().webirc("password", "user", "host", ip).build();
 ```
+
+### Further reading
+
+* [Mibbit WebIRC page](https://wiki.mibbit.com/index.php/WebIRC) - covers the structure of the WEBIRC command, expectatoions for clients and servers supporting the protocol and more.
```
Comment (mbax) [inline docs/advanced/webirc.md]: > expectatoions

File: docs/advanced/webirc.md (hunk 1 above)
Comment (lol768) [inline docs/advanced/webirc.md]: Fixed.

File: docs/advanced/webirc.md (hunk 2 above)
Comment (lol768) [inline docs/advanced/webirc.md]: discussed on irc

**Comment by lol768** [comment]
https://korobi.io/network/esper/channel/kitteh.org/logs/2015/10/09/#L170

**Comment by lol768** [comment]
Above comment in IRC should now be addressed.

File: docs/advanced/webirc.md [hunk 4]
```diff
@@ -1,6 +1,21 @@
-KICL supports the WEBIRC command.
+*Please note that some server software refers to the WebIRC protocol as cgiirc in configuration files.*
+
+KICL supports the WEBIRC command which is used for indirect connections such that the IRC server is made aware of the user's own IP address. For example, consider a webchat client. Without WebIRC support, all connections made through the web client would appear to originate fromt the webserver's IP address. With WebIRC support, the (trusted) webchat client sends the following information on connect:
```
Comment (mbax) [inline docs/advanced/webirc.md]: fromt

**Comment by mbax** [comment]
https://github.com/KittehOrg/KittehIRCClientLib/commit/1259500310e9f0671795545dbbcbcbb384df3e61 thanks

**Pull request closed without merging**